[#93705874] OpenConnect ansible role.

mtekel opened the pull request:
[Create an authentication mechanism for the build service](https://www.pivotaltracker.com/story/show/93705874)

**What**

Create a simple ansible role to install and configure OpenConnect SSL VPN client.

ajlanghorn commented on meta/main.yml:
```diff
@@ -0,0 +1,130 @@
+---
+galaxy_info:
```
This whole section probably needs completing appropriately if it's going to get submitted to Galaxy, or perhaps removed if it's not.

actionjack committed:
Add MIT license file

actionjack committed:
Create ansible role to install openconnect

Create an ansible role to install, configure and manage `openconnect`
ssl vpn client.

**Configured with the following default parameters:**

* `dnsupdate: true` -  Update your resolvers as instructed by the VPN server

actionjack committed:
Add README on playbook usage

actionjack committed:
Add ansible-galaxy meta file

actionjack commented:
@ajlanghorn We've updated the meta file.

ajlanghorn commented:
Great, thanks @actionjack. I would also consider changing the `author` value to `Government Digital Service`, or similar.

multicloud-user commented:
Ah yes :+1: on author to GDS, thanks @ajlanghorn.

multicloud-user commented:
NB I have not reviewed this, just weighing in to support Andrew's comment.

actionjack commented:
@ajlanghorn @multicloud-user Author updated

multicloud-user commented:
Sorry - that was Anna, logged in as multicloud-user :)

ajlanghorn commented:
The only other thing I see is that 'company' value being set to 'Government Digital Service' or 'HM Government' or something so that you set the precedent now. But that's me being possibly a bit picky?

actionjack commented:
@ajlanghorn Fixed!

ajlanghorn commented:
@actionjack Cool. I'll leave a responsible adult to merge. Not sure I qualify ;)

jimconner commented:
LGTM - Merging..
